tests/linearGaussian_snl_test.py

```diff
@@ -65,7 +63,7 @@ def test_snl_on_linearGaussian_based_on_mmd(num_dim: int, prior_str: str, set_se
     """
 
     x_o = zeros((1, num_dim))
-    num_samples = 200
+    num_samples = 1000
```

Comment:
Sampling is really really slow for SNL. Let's please keep this number as low as possible to now have our test time blow up. Does the classifier really need this many samples?

Reply:
OK, I will have to experiment with c2st a bit then, to check for how many samples the classifier is reliable.

Reply:
OK, for a 2D Gaussian linear with fixed variance c2st can detect a difference in means of 0.15 with about 200 samples (score>0.6). Thus we should be fine with 200 samples. Or to be on the save side, let's do 300. Because we really dont want to underfit the classifier

Reply:
Ah, that's great news! Yes, let's do 300.